commit ef166b7027e2eb25812b81721c0e3dcd7db76562 (refs/changes/52/352/1)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-08-25T23:05:15+00:00 (3 years, 1 month ago)
    
    OS-5628 mdata-client needs to do its check_rtime part

diff --git a/check_rtime.ex b/check_rtime.ex
new file mode 100644
index 0000000..bf0fbad
--- /dev/null
+++ b/check_rtime.ex
@@ -0,0 +1,10 @@
+#
+# Exceptions for check_rtime logic. This list should be whittled down
+# over time. For an explanation, see check_rtime(1ONBLD) or
+# exception_lists/check_rtime in illumos-joyent.
+#
+
+SKIP	^usr/sbin/mdata-delete$
+SKIP	^usr/sbin/mdata-get$
+SKIP	^usr/sbin/mdata-list$
+SKIP	^usr/sbin/mdata-put$
